### ISSUE TITLE
lsyncd package links man page to incorrect location

### ISSUE THREAD

**Issue by anthonyclarka2** [Homebrew/homebrew-core#7709]
After "brew install lsyncd"
`
/usr/local/Cellar/lsyncd/2.1.6/bin/lsyncd
/usr/local/Cellar/lsyncd/2.1.6/share/man/lsyncd.1

$ pwd
/usr/local/share/man

$ ls
fr           fr.UTF-8     it.ISO8859-1 ja           man1         man4         man7         pl           pl.UTF-8     ru.UTF-8
fr.ISO8859-1 it           it.UTF-8     lsyncd.1     man3         man5         man8         pl.ISO8859-2 ru.KOI8-R    whatis

$ ls -al lsyncd.1
lsyncd.1 -> ../../Cellar/lsyncd/2.1.6/share/man/lsyncd.1

$ man lsyncd
No manual entry for lsyncd

$ cd /usr/local/share/man/man1

$ ln -sf /usr/local/Cellar/lsyncd/2.1.6/share/man/lsyncd.1

$ man lsyncd
... man output works OK ...
`

short version:  /usr/local/Cellar/lsyncd/2.1.6/share/man/lsyncd.1 is currently soft linked to /usr/local/share/man/lsyncd.1.  It needs to be linked to /usr/local/share/man/man1/lsyncd.1

**Comment by MikeMcQuaid** [comment]
Thanks for the great explanation! Could you try and open a pull request? [This document](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged) should help and I'm happy to walk you through anything else.

Thanks!

**Comment by ilovezfs** [comment]
The formula does this:
```
    # DESTINATION man
    inreplace "CMakeLists.txt", "DESTINATION man",
                                "DESTINATION #{man}"
```
@anthonyclarka2 so in your PR, you can make that `"#{man1}"` instead of `#{man}` and it should be fixed.

**Comment by anthonyclarka2** [comment]
my apologies, I didn't see this get updated from when I created this issue.  I would have created a PR and contributed a fix.

**Comment by ilovezfs** [comment]
@anthonyclarka2 no problem. Thanks for reporting it!
